File: 프로그래머스/0/181935. 홀짝에 따라 다른 값 반환하기/홀짝에 따라 다른 값 반환하기.py
def solution(n):
    answer = 0
    tmp = n
    
    if n % 2 == 0 :
        while tmp > 0 :
            answer += tmp * tmp
            tmp -= 2
    
    elif  n % 2 == 1 :
        while tmp > 0 :
            answer += tmp
            tmp -= 2
            
    return answer

File: 프로그래머스/0/181935. 홀짝에 따라 다른 값 반환하기/홀짝에 따라 다른 값 반환하기.cpp
#include <string>
#include <vector>

using namespace std;

int solution(int n) {
    int answer = 0;
    int tmp = n;
    
    if (n % 2 == 0)
        while (tmp > 0)
        {
            answer += tmp * tmp;
            tmp -= 2;
        }

    else if  (n % 2 == 1)
        while (tmp > 0)
        {
            answer += tmp;
            tmp -= 2;
        }   
                
    return answer;
}